kasa/smartdevice.py

```diff
@@ -411,11 +424,13 @@ def has_children(self) -> bool:
         return self.is_strip
 
     @property  # type: ignore
-    @requires_update
+    @requires_discovery_or_update
```

Comment:
Let's not add any more complexity to the mix, but rather keep using `@requires_update` for these.

In most of the cases this is rather superfluous anyway.

Reply:
Removed